distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc

```diff
@@ -1,48 +1,83 @@
 
-====== NITF-specific Attributes Provided by ${cal-branding}
+====== Attributes Provided by ${cal-branding}
 
-${cal-branding} includes support for accessing additional metadata from specific file formats, primarily NITF. In addition to the above attributes, ${branding} can extract these attributes from NITF resources, if populated:
+${cal-branding} includes support for accessing additional metadata from ISR specific data formats. 
 
-.${cal-branding} Additional NITF Attributes
+.${cal-branding} supported data formats:
+* MPEG2/4 Video Transport Streams including MISB compliant KLV data
+* Image data formats - NITF 2.0,2.1 / NSIF (STANG 4545) 
+* Moving Target Indicator (MTI) (STANAG 4607)
+* STANAG 4559 - NSILI  
+ 
+.${cal-branding} Additional Attributes
 
-${branding} has the capability to extract many NITF-specific Attributes from NITF files. (available if populated)
+${branding} has the capability to extract ISR-specific metadata from supported file types:
 
-* General Attributes
-** `location`
-** `title`
-** `created`
-** `modified`
-** `effective`
+* Security Attributes
 ** `security.classification`
 ** `security.classification-system`
 ** `security.codewords`
 ** `security.dissemination-controls`
 ** `security.releasability`
-** `contact.creator-name`
-** `contact.creator-phone`
-** `isr.mission-id`
-** `isr.niirs`
+
+* General ISR Attributes
+
+** `isr.frequency-hertz`
```

Comment:
Are these in any particular order? I wonder if we should consider alphabetical in the future.